install/wavefront/templates/wavefront.yaml

```diff
@@ -4,11 +4,11 @@ apiVersion: "config.istio.io/v1alpha2"
 kind: adapter
 metadata:
   name: wavefront
-  namespace: istio-system
+  namespace: {{ .Values.namespaces.istio }}
 spec:
   description: 
   session_based: false
   templates:
-  - metric
+  - metric.{{ .Values.namespaces.adapter }}
```

Comment:
Could you document the need to update this file in the `BUILD` file? This file is auto-generated by the mixer codegen utility, and we now need to update the namespace sections as a post-generation step.

Ideally, we should revert the change to this file and the `namespaces.istio` value and we should only take the `namespaces.adapter` value as input. But, that's for another day as it requires an additional debugging/update of Istio dependencies.